src/psyclone/tests/test_files/dynamo0p3/infrastructure/README.md

```diff
@@ -33,3 +33,11 @@ will add support for reading in meshes using NetCDF in a library
 ``liblfric_netcdf.a``. Access to ``nf-config`` is required to
 detect the required settings. When this library is used you
 have to link in NetCDF after ``liblfric_netcdf.a``.
+
+## Using the Intel Compiler
+In order to use the Intel compiler, use the F90 and F90FLAGS
```

Comment:
Again, I'd suggest using code quotes for the compiler flags, ``F90`` and ``F90FLAGS``.

Reply:
Done.

Reply:
It doesn't seem so :smiley:

Reply:
Not quite :smiley:

Reply:
Same issue, push didn't come through.